src/ol/layer/MapboxVector.js

```diff
@@ -197,9 +197,11 @@ const SourceType = {
  * @property {number} [minResolution] The minimum resolution (inclusive) at which this layer will be
  * visible.
  * @property {number} [maxResolution] The maximum resolution (exclusive) below which this layer will
- * be visible.
+ * be visible. If neither `maxResolution` nor `minZoom` are defined, the layer's `maxResolution` will
+ * match the source's `minZoom`
  * @property {number} [minZoom] The minimum view zoom level (exclusive) above which this layer will be
- * visible.
+ * visible. If neither `maxResolution` nor `minZoom` are defined, the layer's `maxResolution` will
```

Comment:
```suggestion
 * visible. If neither `maxResolution` nor `minZoom` are defined, the layer's `minZoom` will
```
maybe?  seems fine either way